airflow/contrib/hooks/gcp_text_to_speech_hook.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from google.cloud.texttospeech_v1 import TextToSpeechClient
+
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+class GCPTextToSpeechHook(GoogleCloudBaseHook):
+    """
+    Hook for Google Cloud Text to Speech API.
+
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :type gcp_conn_id: str
+    :param delegate_to: The account to impersonate, if any.
+        For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+    :type delegate_to: str
+    """
+
+    _client = None
+
+    def __init__(self, gcp_conn_id="google_cloud_default", delegate_to=None):
+        super(GCPTextToSpeechHook, self).__init__(gcp_conn_id, delegate_to)
+
+    def get_conn(self):
+        """
+        Retrieves connection to Cloud Text to Speech.
+
+        :return: Google Cloud Text to Speech client object.
+        :rtype: google.cloud.texttospeech_v1.TextToSpeechClient
+        """
+        if not self._client:
+            self._client = TextToSpeechClient(credentials=self._get_credentials())
+        return self._client
+
+    def synthesize_speech(self, input_, voice, audio_config, retry=None, timeout=None):
+        """
+        Synthesizes text input
+
+        :param input_: text input to be synthesized. See more:
```

Comment:
really hate this kind of param names (ending in underscore) .. can we change `input_` to something else?

Reply:
I will change it to `input_data`